src/routes/+layout.svelte

```diff
@@ -105,5 +107,27 @@
 	</svelte:fragment>
 	<!-- Page Route Content -->
 	<slot />
-	<svelte:fragment slot="footer">Page Footer</svelte:fragment>
+	<svelte:fragment slot="pageFooter">
+			<div class="grid grid-cols-2 place-items-center md:!grid-cols-4 gap-1 lg:ml-10 lg:mr-10">
+
+				<enhanced:img
+					src={FooterImg1}
+					alt="Ulrich Becher in Basel"
+				/>
+
+				<enhanced:img
+					src={FooterImg2}
+					alt="Ulrich Becher in Basel"
+				/>
+
+				<p class="h5 md:h6 lg:h5 text-black">
```

Comment:
```suggestion
				<p class="h5 md:h6 lg:h5 text-black col-span-2 justify-self-start">
```
As [I mentioned](https://github.com/DHBern/becher-frontend/pull/4#discussion_r1470227513) I wanted the text to span two cols.